wgpu-hal/src/vulkan/instance.rs

```diff
@@ -786,7 +786,7 @@ impl crate::Surface<super::Api> for super::Surface {
             index,
             texture: super::Texture {
                 raw: sc.images[index as usize],
-                drop_guard: None,
+                externally_owned: false,
```

Comment:
This is confusing to me.
- In the case of `Instance`, you replaced a `drop_guard` argument of `Some(Box::new(()))` with an `externally_owned` argument of `false`.
- In the case of `Texture`, you replaced a `drop_guard` value of `None` with an `externally_owned` argument of `false`.

Did the presence of a `DropGuard` really mean "externally owned" in `Instance` and "not externally owned" in `Texture`?